packages/@aws-cdk/aws-servicecatalog/lib/template.ts

```diff
@@ -0,0 +1,93 @@
+import * as s3_assets from '@aws-cdk/aws-s3-assets';
+import * as cdk from '@aws-cdk/core';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from '@aws-cdk/core';
+
+/**
+ * Represents the Product Provisioning Artifact Template.
+ */
+export abstract class Template {
```

Comment:
Template is a very generic term... I wonder if `CloudFormationTemplate` is not a better name here?